app/Domain/Tickets/Repositories/Tickets.php

```diff
@@ -421,27 +417,22 @@ public function getAllBySearchCriteria(array $searchCriteria, string $sort = 'st
                     t2.profileId AS editorProfileId,
                     milestone.headline AS milestoneHeadline,
                     IF((milestone.tags IS NULL OR milestone.tags = ''), 'var(--grey)', milestone.tags) AS milestoneColor,
-                    COUNT(DISTINCT zp_comment.id) AS commentCount,
-                    COUNT(DISTINCT zp_file.id) AS fileCount,
-                    COUNT(DISTINCT subtasks.id) AS subtaskCount,
+                    (SELECT COUNT(*) FROM zp_comment WHERE zp_tickets.id = zp_comment.moduleId and zp_comment.module = 'ticket') AS commentCount,
```

Comment:
This is okay for now but moving forward we should remove those calls in favor of async htmx calls. I have done something similar with the project hub overview page where the progress calculation comes separately as htmx request.